bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WorkbenchPlugin.java

```diff
@@ -766,6 +774,10 @@ public void start(BundleContext context) throws Exception {
 			// to be loaded.s
 			if (uiBundle != null)
 				uiBundle.start(Bundle.START_TRANSIENT);
+
+			List<String> recentlyUsedNewWizards = recentNewWizardsPreferenceManager.getMenuShortcutsFromPreferences();
```

Comment:
This code doesn't belong to the bundle activator, it is an internal implementation detail of the RecentNewWizardsPreferenceManager and should be done there.